prog/bootstrap_rhizome.rb

```diff
@@ -20,7 +20,7 @@ def user
     key_data = sshable.keys.map(&:private_key)
     Util.rootish_ssh(sshable.host, user, key_data, <<SH)
 set -ueo pipefail
-sudo apt update && sudo apt-get -y install ruby-bundler
+sudo apt update && sudo apt-get -y install ruby-bundler daemonize
```

Comment:
I would not grow another dependency on daemonize, use `systemd-run`

Reply:
I used `systemd-run` without `--no-block`. According to man page `systemd-run` already runs the command asynchronously; passing `--no-block` additionally skips waiting for start up.

> ... systemd-run will start the service asynchronously in the background and return after the command has begun execution (unless --no-block or --wait are specified, see below).
>
> ...
>
> --no-block
> Do not synchronously wait for the unit start operation to finish. If this option is not specified, the start request for the transient unit will be verified, enqueued and systemd-run will wait until the unit's start-up is completed. By passing this argument, it is only verified and enqueued. This option may not be combined with --wait.

I prefer waiting for start up, because it eliminates the risk of stdin file to be changed. In my experiments, if stdin file changes after start of process, process still sees old version of it. But this is me experimenting; I couldn't find an authoritative source that describes edge cases.

I do status reporting with `systemctl show -p SubState --value #{name}.service`. I found this to be most comprehensive because it returns 4 different values for 4 different cases we care about (not started, in progress, succeeded, failed). It also has other values that we don't use (can be found via `systemctl --state=help`, under the "Available service unit substates")

I do stdout/stderr capture by redirecting them to particular files. What do you mean by `journalctl` integration via stdout/stderr capture?

Reply:
Resolving for now. We can discuss the details later on.